src/transform.js

```diff
@@ -54,7 +54,7 @@ export default function transform(strings, values) {
   let mergeWithLastString = false;
   values.forEach((value, index) => {
     const string = strings[index];
-    if (value.prototype instanceof HTMLElement) {
+    if (value && value.prototype instanceof HTMLElement) {
```

Comment:
LGTM, good find!

Reply:
Good idea, this will automatically happen if you "Squash commits" upon merging the pull requests.